apps/alerting/notifications/kinds/v0alpha1/routingtree_spec.cue

```diff
@@ -0,0 +1,33 @@
+package v0alpha1
+
+RouteTreeSpec: {
+	defaults: #RouteDefaults
+	routes: [...#Route]
+}
+
+#RouteDefaults: {
+	receiver: string
+	group_by?: [...string]
+	group_wait?:      string
+	group_interval?:  string
+	repeat_interval?: string
+}
+
+#Matcher: {
+	type:  "=" | "!=" | "=~" | "!~" @cuetsy(kind="enum",memberNames="Equal|NotEqual|EqualRegex|NotEqualRegex")
```

Comment:
using `memberNames` we can control the output var names so we don't need to alias in `ext.go`